wordpress-mobile/gutenberg-mobile
# Fix unintended home button trigger on iOS in e2e tests

gutenberg PR: https://github.com/WordPress/gutenberg/pull/27858

To test: 

Locally: `npm run test:e2e:ios:local gutenberg-editor-latest-posts`
CI: Check if all e2e tests pass

PR submission checklist:
